### ISSUE TITLE
How Do the Hearts of Deep Fakes Beat? Deep Fake Source Detection via Interpreting Residuals with Biological Signals

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#107]
# TL;DR
DeepFake (fake video) using deep generative models has become a social problem, and They achieved 97.29% accuracy in DeepFake detection by using information from a biological signal called PPG. They also found that each generation model generates a unique false PPG-like signal, and were able to identify the generation model with 93.39%.
![Figure 2  PPG Cells  Example frames per w = 64 window (top), and their PPG cells (bottom) consisting of raw PPG and PPG PSD, of a](https://user-images.githubusercontent.com/24524018/93066647-36600f80-f6b5-11ea-8606-800d9e115005.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.11363

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Umur Aybars Ciftci, Ilke Demir, Lijun Yin
- Binghamton University
- Intel

# Methods

# Results

# Comments

**Comment by nineLemon9** [comment]
Hello，do you know where can I get the code for this paper？
